app/src/main/cpp/skyline/gpu/interconnect/graphics_context.h

```diff
@@ -974,6 +1018,17 @@ namespace skyline::gpu::interconnect {
                         executor.AttachDependency(std::move(sampler));
                     }
                 }
+                if (!program.info.image_buffer_descriptors.empty()) {
```

Comment:
No {} round one line ifs, also shouldn't this be debug

Reply:
It originally was but I figured, it's probably not the worst idea to keep that in there.

Reply:
Nor sure what you mean here? It was originally debug or originally {}?